packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -58,7 +58,11 @@ export class CsvGenerator {
     private stream: Writable
   ) {}
 
-  private async openPointInTime(indexPatternTitle: string, settings: CsvExportSettings) {
+  private async openPointInTime(
+    indexPatternTitle: string,
+    settings: CsvExportSettings,
+    maxConcurrentShardRequests: number
+  ) {
     const { duration } = settings.scroll;
```

Comment:
We don't need the 3rd parameter, since `maxConcurrentShardRequests` is available in `settings`.

I recommend getting all we need from `settings` like:
```typescript
    const {
      includeFrozen,
      maxConcurrentShardSize,
      scroll: { duration },
    } = settings;
```